README.md

```diff
@@ -19,18 +19,18 @@ Use Composer to install the library:
 composer require knplabs/knp-time-bundle
 ```
 
-Woo! You did it! Assuminy project uses Symfony Flex, the
+Woo! You did it! Assuming your project uses Symfony Flex, the
 bundle should be configured and ready to go. If not, you
 can enable `Knp\Bundle\TimeBundle\KnpTimeBundle` manually.
 
 ## USAGE
 
-In Twig!
+In Twig:
 
-```html+jinja
+```twig
 {{ someDateTimeVariable|ago }}
 
-... or use the equivalent function
+{# ... or use the equivalent function: #}
```

Comment:
```suggestion
... or use the equivalent function:
```

Reply:
I think the twig comment tags take away from clarity of the intention. It took me a second to figure out what we're doing here...